ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -21,9 +21,11 @@ public class BottomSheetController: UIViewController {
     /// - Parameters:
     ///   - headerContentView: Top part of the sheet content that is visible in both collapsed and expanded state.
     ///   - expandedContentView: Sheet content below the header which is only visible when the sheet is expanded.
-    @objc public init(headerContentView: UIView? = nil, expandedContentView: UIView) {
+    ///   - dimsMainContent - Indicates if the main content is dimmed when the sheet is expanded.
```

Comment:
misspelled

Reply:
Can you be more specific? I can't find the typo. 😄

Reply:
sorry my bad. not a typo but i think the name sould be like shouldShowDimmingView? or something more swift like?

Reply:
ohh I like that! I spent SO much time trying to find a swifty name and I haven't thought of that one.